bigipreport.ps1

```diff
@@ -1537,14 +1547,15 @@ function Get-LTMInformation {
         $ObjTempVirtualServer.availability = $VirtualServerStates[$i].availability_status
         $ObjTempVirtualServer.enabled = $VirtualServerStates[$i].enabled_status
 
+        Try {
+	# FIXME: 15.x fails to load vittual server statistics
```

Comment:
I assume vittual should be virtual?

Reply:
Are you fixing the spelling or should I?

Reply:
I got it. :)